x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -269,9 +282,16 @@ protected boolean supportsInferenceTestService() {
         return true;
     }
 
+    protected boolean supportsInferenceTestServiceOnLocalCluster() {
+        return supportsInferenceTestService();
+    }
+
     protected boolean requiresInferenceEndpoint() {
```

Comment:
let's rename this to `requiresSemanticTextInference` - the same for `supportsInferenceTestService` -> `supportsSemanticTextInference`.
because it's hard to figure out the difference from `requiresInferenceEndpointOnLocalCluster` vs `requiresInferenceEndpoint` without looking at the implementation.